### PR TITLE
feat(SC-1110): Added code to update spreadsheet output for static mod…

### PR DESCRIPTION
**Pull request by abnepc**
## Jira Story
[SC-1110](https://epcpower.atlassian.net/browse/SC-1110)
## About
Added scale factor column and enumerations worksheet to static modbus spreadsheet output
## Associated Pull Requests
*   [PR-8](https://github.com/epcpower/rm-dc/pull/8)

## Update Changelog

*   [x] Changelog updated

## Reproduction Steps
N/A
## Validation
Follow the steps for validation in the associated PR

[SC-1110]: https://epcpower.atlassian.net/browse/SC-1110?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/epcpm/pm_helper.py
Comment (gordon-epc) [inline src/epcpm/pm_helper.py]: This will break for regular SunSpec. Need to leave `sunssf` in.
```suggestion
        if type_node.name == "sunssf" || type_node.name == "staticmodbussf":
```
Unless there is a more clever way to do this?

File: src/epcpm/pm_helper.py
Comment (gordon-epc) [inline src/epcpm/pm_helper.py]: The `build_uuid_scale_factor_dict` method still needs to work as it did before. Almost like you will have to merge the before and after.

File: src/epcpm/staticmodbustoxls.py
Comment (gordon-epc) [inline src/epcpm/staticmodbustoxls.py]:
```suggestion
```

**Review by gordon-epc — CHANGES_REQUESTED** [review]

File: src/epcpm/staticmodbustoxls.py
Comment (abnepc) [reply src/epcpm/staticmodbustoxls.py]: [773121a](https://github.com/epcpower/pm/pull/72/commits/773121a56640e4204d381751c10068cf322c7371)

File: src/epcpm/pm_helper.py
Comment (abnepc) [reply src/epcpm/pm_helper.py]: [773121a](https://github.com/epcpower/pm/pull/72/commits/773121a56640e4204d381751c10068cf322c7371)

**Comment by gordon-epc** [comment]
Waiting on approval from R100 folks.

File: src/epcpm/pm_helper.py
Comment (gordon-epc) [inline src/epcpm/pm_helper.py]:
```suggestion
        DataPointBitField objects from which to generate scale factor data
```

**Review by gordon-epc — CHANGES_REQUESTED** [review]

File: src/epcpm/pm_helper.py
Comment (abnepc) [reply src/epcpm/pm_helper.py]: [80b4a94](https://github.com/epcpower/pm/pull/72/commits/80b4a940f81c1ab04f2cfc999c2bc6c51ee3f556)

**Review by gordon-epc — APPROVED** [review]

**Pull request merged**
